### ISSUE TITLE
‘Choose NuGet Package Manager Format’ dialog won’t pop out again after canceling installation from ‘Preview’ window with ‘PR’ format selected

### ISSUE THREAD

**Issue by v-luzh** [NuGet/Home#4946]
**NuGet Version:** NuGet 4.3.0-Beta1-2447 (Insertable)
**VS Version:** VSUWPT 26330.00
**OS Version:** Windows 10 RS2_Release 15063.2.170323-1837

**Repro Steps:**
1. Check **‘Allow format selection on first package install’** in NuGet Package Manager setting dialog.
2. Restart VS and create a C# Class Library (.NET Framework).
3. Open the ‘Manage NuGet Packages…’ for the project.
4. Go to ‘Browse’ tab, click the package ‘Newtonsoft.Json’ and click ‘Install’ button in the right panel.
5. Select **‘PackageReference in project file’** format and click OK button.
6. Click **‘Cancel’** button in the following ‘Preview’ window.
7. Redo step 4.

**Expected Result:**
The ‘Choose NuGet Package Manager Format’ dialog should pop out again (like what we have if you selected ‘Packages.config’ in step 5) since the first installation is not complete yet.

**Actual Result:**
The ‘Choose NuGet Package Manager Format’ dialog didn’t pop out again as below screenshot.
![differencebetweenpcandpr](https://cloud.githubusercontent.com/assets/20632625/24545789/3a543e1c-163b-11e7-89c5-2bdcd97da6c8.png)

**Note:**
1.It repro on NuGet Client 4.1.0-RTM-2427.
2.It also repro on D15Rel 26330.00 + NuGet Client 4.1.0-RTM-2445 Insertable.

**Comment by jainaashish** [comment]
It can go both ways. Since you already made up your mind to go with PackageReference, and selected it. And then why should it show the dialog again?

Will discuss it with PM for more clarity. Either way, it should be low priority and doesn't feel like a real customer scenario.

**Comment by v-luzh** [comment]
@jainaashish Yes, the priority is low. But we got different results from 'Package.config' and 'PackageReference' formats. In my opinion, the 'Package.config' is better since it provide the user a chance to reselect if he/she clicked the wrong format in step5. :)

**Comment by v-luzh** [comment]
It still reproes on VS Main\35221.199 + NuGet Client Dev\6.12.0.81.

**Comment by jgonz120** [comment]
@JonDouglas this issue is quite old, but do you feel like this experience needs to be changed? My expectation would be once I select the format I wouldn't get the pop-up again, regardless of if I completed install. 

If the install resulted in an error, meaning an install wasn't technically completed, I wouldn't expect to see this dialog again.

**Comment by nkolev92** [comment]
Closing as low impact, won't fix.
